src/Feature/Layout/Feature_Layout.re

```diff
@@ -0,0 +1,284 @@
+open Utility;
+
+type direction =
+  | Up
+  | Left
+  | Down
+  | Right;
+
+type t('content) =
+  | Split([ | `Horizontal | `Vertical], list(t('content)))
+  | Window({
+      weight: float,
+      content: 'content,
+    })
+  | Empty;
```

Comment:
The window is now called a window instead of a split, and the split is called a split instead of "parent". Otherwise this is pretty much the same tree structure as before.

Reply:
Sounds good 👍